samples/Demo Data/Assets/Scripts/CategotySceneBehavior.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using Datadog.Unity;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class CategotySceneBehavior : MonoBehaviour
+{
+
+    // Start is called before the first frame update
```

Comment:
Missing copyright header and typo in the class name: `Categoty`. It would also make sense to call it `Behaviour` with the "u" to be consistent with Unity naming.